src/Tests/Microsoft.NET.Build.Tests/GivenWeWantToRequireWindowsForDesktopApps.cs

```diff
@@ -228,6 +228,33 @@ void Assert(DirectoryInfo outputDir)
                 .Pass();
 
             Assert(publishCommand.GetOutputDirectory(tfm, runtimeIdentifier: runtimeIdentifier));
+
+            var command = new GetValuesCommand(
+                Log,
+                Path.Combine(asset.Path, testProject.Name),
+                testProject.TargetFrameworks,
+                "FilesCopiedToPublishDir",
+                GetValuesCommand.ValueType.Item)
+            {
+                DependsOnTargets = "ComputeFilesCopiedToPublishDir",
+                MetadataNames = { "RelativePath", "IsKeyOutput" },
```

Comment:
What is IsKeyOutput?

Reply:
I don't know exactly. But only the exe has it. @sbanni do you know?

Reply:
The test isn't reading the value of the metadata though.  So it seems like either the test should assert something about IsKeyOutput, or it shouldn't need to be in the MetadataNames list.

Reply:
I am removing MetadataNames IsKeyOutput

Reply:
the set up is copied from another wapproj test. No need to assert IsKeyOutput here

Reply:
Removed